mmv1/templates/terraform/examples/container_attached_cluster_full.tf.erb

```diff
@@ -0,0 +1,34 @@
+data "google_project" "project" {
+}
+
+resource "google_container_attached_cluster" "primary" {
+  name     = "<%= ctx[:vars]['name'] %>"
+  description = "Test cluster"
+  distribution = "aks"
+  annotations = {
+    label-one = "value-one"
+  }
+  authorization {
+    admin_users = [ "user1@example.com", "user2@example.com"]
+  }
+  oidc_config {
+      issuer_url = "https://oidc.issuer.url"
+      jwks = base64encode("{\"keys\":[{\"use\":\"sig\",\"kty\":\"RSA\",\"kid\":\"testid\",\"alg\":\"RS256\",\"n\":\"somedata\",\"e\":\"AQAB\"}]}")
+  }
+  platform_version = "1.24.0-gke.1"
```

Comment:
This may already be the plan - once the datasource is available, it would be great to replace this with a reference to the datasource.

Reply:
That is the plan. One of the added datasources retrieves all valid versions from the service, so I will replace the hard-coded ones in the tests.